app/_common/components/NavigationBottom.tsx

```diff
@@ -1,12 +1,32 @@
 "use client";
 
+import { MouseEvent } from "react";
 import { usePathname } from "next/navigation";
 
+import { toast } from "../utils/toast";
 import { navigate } from "../utils/redirect";
+import { useAuthStore } from "../store/useAuthStore";
+import { cn } from "../shadcn/utils";
 import { Menubar, MenubarMenu, MenubarTrigger } from "../shadcn/ui/menubar";
+import WithOnMounted from "../hoc/WithOnMounted";
 
 function NavigationBottom() {
   const pathname = usePathname();
+  const { getUser } = useAuthStore();
+
+  const handleNavigateClick = (event: MouseEvent<HTMLButtonElement>) => {
+    const path = (event.target as HTMLButtonElement).value;
+    navigate(path);
+  };
+
+  const handleNoAuthClick = () => {
+    toast.warning("로그인 후 이용해주세요!", {
+      action: {
+        label: "로그인하기",
+        onClick: () => navigate("login"),
+      },
+    });
+  };
 
   return (
     <Menubar
```

Comment:
[전에 말씀드렸던 방법](https://github.com/Open-Eye-Im-Developer/MoGakGo-FE/pull/240#pullrequestreview-1946271090)에 추가적으로 아래처럼 상위 요소에 `onClick`을 걸어주시면 비로그인 안내 토스트는 잘 뜨면서 하이라이트는 안옮겨지도록 할 수 있는 것 같아요.

```suggestion
    <Menubar
      onClick={!getUser() ? handleNoAuthClick : () => {}}
```

Reply:
오 이런 방법이 있었군요!! 감사합니다 잘 작동하는 것 같아요 :D 👍